### DISCUSSION TITLE
JSON failure

### DISCUSSION THREAD

**Discussion by jayvdb** [lowlighter/metrics#607] [Q&A] [answered]
I added an `output_format: json` step to my CI, and it has failed three times
https://github.com/jayvdb/jayvdb/commit/0969aafdabf1ba3d94b3c7f58ed26eaa230c2a58
It is simpler than the SVG settings, so it is a bit surprising that it failed when the SVG was happily building.

When I remove 'indepth', I get a different error

> Error: Converting circular structure to JSON
>    --> starting at object with constructor 'Object'
>    |     property 'question' -> object with constructor 'Object'
>    --- property 'answer' closes the circle
> Error: Process completed with exit code 1.

Removing StackOverflow fixed that.

**Comment by lowlighter** [comment]
There is probably a circular reference somewhere which is why `JSON.stringify` fails

**Comment by lowlighter** [comment, marked as answer]
This should have been fixed by: 
* #623
